tests/acceptance/10_files/copy_from_this_promiser.cf

```diff
@@ -0,0 +1,85 @@
+#!/var/cfengine/bin/cf-agent -f-
+body common control
+{
+        inputs => { "../default.cf.sub" };
+        bundlesequence  => { default("$(this.promise_filename)") };
+        version => "1.0";
+}
+
+
+bundle agent init {
+  vars:
+      "ROOT" string => "/tmp/CFE-2489";
+      "SOURCE" string => "$(ROOT)/SOURCE";
+      "use_bundles"
+        slist => { "bundle_uses_copy_from_without_param",
+                   "bundle_uses_copy_from_with_param" };
+
+  files:
+      "$(init.ROOT)/$(use_bundles)/." create => "true";
```

Comment:
why init.ROOT aren't you still in the same bundle so just ROOT would work?

Reply:
The only reason was so that it was the same in other places making it easier for me to understand what was going on.

Reply:
yes, ROOT would work.